src/Tribe/Assets.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Handles registering all Assets for the Events Virtual Plugin.
+ *
+ * To remove a Asset you can use the global assets handler:
+ *
+ * ```php
+ *  tribe( 'assets' )->remove( 'asset-name' );
+ * ```
+ *
+ * @since 1.1.0
+ *
+ * @package Tribe\Events\Virtual
+ */
+namespace Tribe\Extensions\Event_Link_To_Website_URL;
+
+/**
+ * Register Assets.
+ *
+ * @since 1.1.0
+ *
+ * @package Tribe\Extensions\Event_Link_To_Website_URL
+ */
+class Assets extends \tad_DI52_ServiceProvider {
```

Comment:
Do we need this class if we're not adding/using assets?

Reply:
I think it was in the extension template, but you're right we don't need it for this one